client/hotrod-client/src/main/java/org/infinispan/client/hotrod/ServerStatistics.java

```diff
@@ -18,17 +18,35 @@
 
    /**
     * Number of entries currently in the Hot Rod server
+    * @deprecated Since 14.0, please use {@link #APPROXIMATE_ENTRIES}
     */
+   @Deprecated
    String CURRENT_NR_OF_ENTRIES = "currentNumberOfEntries";
 
    /**
     * Number of entries stored in Hot Rod server
     * since the server started running.
+    * @deprecated Since 13.0, please use {@link #STORES} instead.
     */
+   @Deprecated
    String TOTAL_NR_OF_ENTRIES = "totalNumberOfEntries";
 
    /**
-    * Number of put operations.
+    * Approximate number of entry copies currently in the server cache, including persistence.
+    *
+    * Only entries local to the server receiving the request are included.
+    */
+   String APPROXIMATE_ENTRIES = "approximateEntries";
+
+   /**
+    * Number of unique entries currently stored in server cache, including persistence.
+    *
+    * Only entries primary-owned by the server receiving the request are included.
+    */
+   String APPROXIMATE_ENTRIES_UNIQUE = "approximateEntriesUnique";
+
+   /**
+    * Number of entries ever stored.
```

Comment:
Yes, all inserts and updates are included. I could copy the javadoc from `TOTAL_ENTRIES` and include the fact that it refers only to the server handling the request, something like "Number of entries stored in the server receiving the request since the server started running."

Reply:
That sounds clearer to me tbh.